app/Http/Controllers/DraftController.php

```diff
@@ -88,7 +88,7 @@ public function getDraftBanData(Request $request){
     "|"  . implode(",", $role_league_tier);
 
     $ban_draft_order_data = Cache::remember($cache, $cache_time, function () use ($game_versions_minor, $game_type, $player_league_tier, $hero_league_tier, $role_league_tier, $game_map, $hero_level, $region){
-      $ban_data = new \GlobalHeroDraftOrder($game_versions_minor, $game_type, $player_league_tier, $hero_league_tier, $role_league_tier, $game_map, $hero_level, $region);
+      $ban_data = new \App\Data\GlobalHeroDraftOrder($game_versions_minor, $game_type, $player_league_tier, $hero_league_tier, $role_league_tier, $game_map, $hero_level, $region);
```

Comment:
So I don't think this needs the \App\Data to be added in front of these as I put a reference in the config/app file.  See below, line 240 I think 


https://github.com/Heroes-Profile/heroesprofile/blob/develop/config/app.php

Reply:
Gotcha. We probably need to supply that file to PHPStan so the alias can be accounted for. If you prefer to use the alias I can change that later tonight.

Reply:
Even if I end up switching up aliasing for these classes later, I still want to be able to use it.  So I would go ahead and add that and then switch the ones you had added App\Data to

Reply:
Actually the underlying issue to this one is that the `GlobalHeroDraftOrder` class already exists in **database/migrations/2020_09_11_174720_global_hero_draft_order.php**. I guess because the controller and the migration aren't normally ever loaded in the same run this has not caused issues, but I would strongly recommend resolving this conflict int he code rather than tinkering with the static analysis.

Reply:
It looks like why this one in particular was an issue is all the other data classes were suffixed "Data". I updated this one accordingly.